v2/azure-arm.yaml

```
@@ -871,6 +871,17 @@ objectModelConfiguration:
       PrivateLinkResource:
         Id:
           $armReference: true
+  datafactory:
+    2018-06-01:
+      PurviewConfiguration:
+        PurviewResourceId:
+          $armReference: true
+      CMKIdentityDefinition:
+        UserAssignedIdentity:
+          $armReference: true
+      Factory:
+        $export: true
+        $supportedFrom: v2.0.0
```

We just released `v2.0.0` so change this to `v2.1.0`.
```suggestion
        $supportedFrom: v2.1.0
```